docs/editing_guidelines.md

```diff
@@ -0,0 +1,138 @@
+>
+> ```
+> Class: 'crystal cell'
+>   SubClassOf: 'hemocyte'
+>   SubClassOf: 'develops from' some 'procrystal cell'
+> ```
+>
+> The first `SubClassOf` axiom denotes an actual classification, whereas the second denotes a relationship.
+
+
+### Asserting classification
+In order to keep the ontology maintainable, asserted classifications should be limited where possible. Ideally, all terms would have only a single asserted parentt. Given the presence of suitable logically defined classes (see below) and sufficient relationships for the term you are making, additional classification can be inferred automatically by reasoning.
+
+However, as we are limited in what types of classification we are able to infer, you may need to assert multiple parents. Two asserted classifications are plainly acceptable. If you must assert three or ore classifications, then you should make a note with a suggestion for which of the asserted classifications are good candidates for formalisation and inferred classification.
+
+
+### Logically defined classes
+Logically defined classes are terms that have formal definitions that specify complete necessary and sufficient conditions for membership of the class. They can be used by a reasoner to auto-classify the ontology by searching for terms that fulfill these conditions.
+
+> Logically defined classes are variously referred to as “cross-products” (XPs), “genus and differentia definitions”, “equivalent classes”, or “intersections”. In OWL formalism, they are represented using `EquivalentClasses` axioms.
+
+Generally, logical definitions follow the structure: “Any `X` that `REL` some `Y` [and `REL` some `Z`...]”, where `X` is the genus and each following clause (“that `REL` some `Y`”) is a differentium.
+
+> Strictly speaking, logical definitions can be arbitrarily complex. It is recommended, however, to stick as much as possible to “simple” logical definitions following the structure above. More complex definitions are harder to understand, and may involve OWL constructs that are legal but not fully supported by the available reasoners.
+
+Good candidates for logically defined classes include classes whose only differentium is:
+
+* what the cell is part of;
+* what it innervates (or, if more precisely known, what its axon(s) or dendrite(s) innervate);
+* its function.
+
+Care must be taken not to use logically defined classes too liberally. You should be satisfied that all automatic classifications that result would make sense (or at least be justifiable) to a biologist and would cover most cases of usages with as few edge cases as possible.
+
+If you are unsure whether a characteristic of a cell should be expressed as a relationship or as a differentium in a logically defined class: use a relationship.
+
+The textual definition of a logically defined class should be a plain english equivalent of the logical definition. However, in some cases it can be useful to add more details in the “gloss” part, when relevant.
```

Comment:
```suggestion
The textual definition of a logically defined class should be a plain English equivalent of the logical definition. However, in some cases it can be useful to add more details in the “gloss” part when relevant.
```